UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs

```diff
@@ -9,7 +9,7 @@ public class CameraSensorComponent : SensorComponent
         public string sensorName = "CameraSensor";
         public int width = 84;
         public int height = 84;
-        public bool grayscale = false;
```

Comment:
I never know which of these suggestions to ignore. For example, it grays out 'private" when it's redundant, but that doesn't seem to be caught here.
https://github.com/Unity-Technologies/ml-agents/blob/58cee7ec0243725c99b573ecc36cf288f7f76975/UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensor.cs#L9-L14

Reply:
this was more about the default value being redundant.  `false` is default for `bool`